server/controllers/stock/index.js

```diff
@@ -336,21 +337,76 @@ function listLotsOrigins(req, res, next) {
  * create a new integration entry
  */
 function createIntegration(req, res, next) {
+  const transaction = db.transaction();
   const params = req.body;
   const identifier = uuid.v4();
+  const documentUuid = uuid.v4();
+  let commonInfos;
+
   const integration = {
     uuid        : db.bid(identifier),
     project_id  : req.session.project.id,
-    description : params.description || 'INTEGRATION',
-    date        : new Date(),
+    description : params.integration.description || 'INTEGRATION',
+    date        : new Date(params.integration.date),
```

Comment:
In some places you use `params.integration.date` and other places you use `params.movement.date`.  Is there a reason for this?  Are they always the same date?

Reply:
When initializing the integration object I used ```params.integration.date```, when inserting the movement object I used ```params.movement.date```. In the commonInfos I have just chosen one of them. SO in theory it is the same date but they are sent by the client so we should be careful and process them as they are provided.

Reply:
Hmm...  in my view, the client should only send one date if they are going to be the same - otherwise it is a [code smell](https://en.wikipedia.org/wiki/Code_smell).